documentation/v&v/V&V.tex

```diff
@@ -221,6 +221,88 @@ \subsection{UserInterface Module (M1)}
 \hline
 M1T19 & book page should allow a user to navigate to the next page of posts & FR5 \\
 \hline
+M1T20 & should allow a user to search and find 5 auto-complete results & FR5, FR6 \\
```

Comment:
I think that maybe M1T20 on should be put onto the next page

Reply:
formatting is fixed